docs/docs/guide/platform/README.md

```diff
@@ -44,50 +44,38 @@ If you're using `react`, DXOS provides a simple [UI flow](../react/spaces#joinin
 
 **Next steps:**
 
-*   Create or join a space in [TypeScript](../typescript/spaces)
-*   Create or join a space in [React](../react/spaces)
+- Create or join a space in [TypeScript](../typescript/spaces)
+- Create or join a space in [React](../react/spaces)
 
 ## Objects
 
-Units of data are referred to as `objects` (like documents or rows in other databases). `Objects` always belong to a space. Objects can have fields with values, and strong relationships to other objects to form trees or graphs.
+Units of data are referred to as `objects` (like documents or rows in other databases). `Objects` always belong to a space. Objects can have fields with values, and weak references to other objects to form trees or graphs.
 
 ## Glossary
 
 See the [glossary](../glossary) for definitions of other terms you'll find in this guide.
 
 ## How to use ECHO
 
-*   Install the appropriate npm package [`@dxos/client`](../typescript) or [`@dxos/react-client`](../react)
-*   Create a [Client](../typescript#configuration) (or a [ClientProvider](../react#cofiguration) in react)
-*   Set up an identity with [HALO](./halo)
-*   Create or Join a [Space](#spaces)
-*   [Query objects](../typescript/queries) (in [react](../react/queries))
-*   [Create objects](../typescript/mutations#creating-objects) (in [react](../react/mutations))
-*   [Mutate objects](../typescript/mutations) (in [react](../react/mutations))
+- Install the appropriate npm package [`@dxos/client`](../typescript) or [`@dxos/react-client`](../react)
+- Create a [Client](../typescript#configuration) (or a [ClientProvider](../react#cofiguration) in react)
+- Set up an identity with [HALO](./halo)
+- Create or Join a [Space](#spaces)
+- [Query objects](../typescript/queries) (in [react](../react/queries))
+- [Create objects](../typescript/mutations#creating-objects) (in [react](../react/mutations))
+- [Mutate objects](../typescript/mutations) (in [react](../react/mutations))
 
-## Local Vault Topology
+## Vault
 
-ECHO is uniquely organized to give control over information to the end-user and their devices.
+ECHO is designed to allow users to own their own data. The ECHO Vault is a secure storage mechanism, responsible for holding end-user identity (keys, credentials, metadata, ..., etc.) and all ECHO data in persistent browser or disk storage. The vault serves as an identity wallet and data vault where specific devices or applications can be revoked from accessing user data at any time.
```

Comment:
ECHO is the database, keys/credentials/etc. are HALO's business.

I think we should just remove all references to "vault" because we're not currently supporting any such mechanism. I'm not sure branding browser storage as "vault" is not super helpful.

Reply:
I'm fine with removing vault everywhere, but I think we need a "branded thing" that refers to the superset of a user's ECHO and HALO. That's how I always thought about vault (not so much the app-spanning across domains technology).